src/main/java/org/broadinstitute/hellbender/utils/bigquery/StorageAPIAvroReader.java

```diff
@@ -54,32 +57,39 @@ public StorageAPIAvroReader(final TableReference tableRef, final String rowRestr
         try {
             logger.info("Using Storage API from " + tableRef.getFQTableName() + " with '" + rowRestriction + "'");
 
-            this.client = BigQueryStorageClient.create();
+            this.client = BigQueryReadClient.create();
 
             final String parent = String.format("projects/%s", parentProjectId == null || parentProjectId.isEmpty() ? tableRef.tableProject : parentProjectId);
 
-            final TableReferenceProto.TableReference tableReference = TableReferenceProto.TableReference.newBuilder()
-                    .setProjectId(tableRef.tableProject)
-                    .setDatasetId(tableRef.tableDataset)
-                    .setTableId(tableRef.tableName)
-                    .build();
+            final String srcTable =
+                    String.format(
```

Comment:
Are there no longer tableReference objects, just table objects? Doing this with a builder pattern feels cleaner